src/helpers.js

```diff
@@ -17,6 +17,11 @@ Marionette.isNodeAttached = function(el) {
   return Backbone.$.contains(document.documentElement, el);
 };
 
+// Merge `keys` from `options` onto `this`
+Marionette.mergeOptions = function(options, keys) {
+  if (!options) { return; }
+  _.extend(this, _.pick(options, keys));
```

Comment:
Yeah, this is already a headache for deeply extended view classes for stuff like `events` and `ui`. They get clobbered really easily.